less/footer.less

```diff
@@ -0,0 +1,80 @@
+@import "variables";
+
+.footer {
+  padding: 100px 0px 300px 0px;
+  color: @color-text-contrast;
+}
+
+.footer__menu {
+  list-style: none;
+  // remove default padding
+  padding: 0px 0px 0px 0px;
+}
+
+.footer__send-link {
+  display: inline-block;
+  padding: 100px 0px 0px 0px;
+}
+
+.footer__link {
+  color: @color-text-contrast;
+
+  &:hover {
+    color: @color-text-contrast;
+  }
+}
+
+.footer__underlined {
+  display: inline-block;
+  border:0;
+  background-color: @color-background-strong;
+  border-bottom:solid 1px @color-text-contrast;
+  outline:none;
+  width: 100%;
+}
+
+.footer__input {
+  padding: 0px 0px 0px 0px;
+  color: @color-text-contrast;
+  width: 100%;
+}
+
+.footer__input_long {
+  padding-top: 20px;
+}
+
+// contains text inputs and send button
+.footer__form {
+  padding: 0px 60px 0px 15px;
+}
+
+.footer__last-row {
+  padding-top: 30px;
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+
+  &::before {
+    position: absolute;
+    content: '';
+    top: 10px;
+    left: 0;
+    width: 100%;
+    height: 1px;
+    background-color: @color-text-contrast;
+  }
+}
+
+.footer__copyright {
+  display: inline-block;
+}
+
+.footer__extra-links {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+}
+
+.footer__extra-link-item {
+  margin: 0px 10px 0px 10px;
+}
```

Comment:
New line https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.